independent-publisher-connectors/Yelp/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1238 @@
+    "version": "1.0",
+    "contact": {
+      "name": "Ahmad Najjar",
+      "email": "ahmadn82@hotmail.com"
+    }
+  },
+  "host": "api.yelp.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v3/businesses/search": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "total": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Total number of business Yelp finds based on the search criteria. Sometimes, the value may exceed 1000. In such case, you still can only get up to 1000 businesses using multiple queries and combinations of the \"limit\" and \"offset\" parameters.",
+                  "title": "Total",
+                  "x-ms-visibility": "advanced"
+                },
+                "businesses": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "rating": {
+                        "type": "number",
+                        "format": "float",
+                        "description": "Rating for this business (value ranges from 1, 1.5, ... 4.5, 5).",
+                        "title": "Rating"
+                      },
+                      "price": {
+                        "type": "string",
+                        "description": "Price level of the business. Value is one of $, $$, $$$ and $$$$.",
+                        "title": "Price",
+                        "x-ms-visibility": "important"
+                      },
+                      "phone": {
+                        "type": "string",
+                        "description": "Phone number of the business.",
+                        "title": "Phone",
+                        "x-ms-visibility": "important"
+                      },
+                      "id": {
+                        "type": "string",
+                        "description": "Unique Yelp ID of this business. Example: '4kMBvIEWPxWkWKFN__8SxQ'",
+                        "title": "ID",
+                        "x-ms-visibility": "important"
+                      },
+                      "alias": {
+                        "type": "string",
+                        "description": "Unique Yelp alias of this business. Can contain unicode characters. Example: \"yelp-san-francisco\"",
+                        "title": "Alias",
+                        "x-ms-visibility": "advanced"
+                      },
+                      "is_closed": {
+                        "type": "boolean",
+                        "description": "Whether business has been (permanently) closed",
+                        "title": "Is Closed",
+                        "enum": [
+                          "",
```

Comment:
@ahmadn82 Can you please let us know how an empty string is allowed in a Boolean enum ?

Reply:
@sriyen-msft I can't TBH... Probably copied it by mistake, as I tend to work on raw swagger file 😉